src/WCSimDetectorConfigs.cc

```diff
@@ -515,18 +515,18 @@ void WCSimDetectorConstruction::SetNuPrismShort_mPMTGeometry()
 	vessel_cyl_height = 38.*CLHEP::mm;    //option A, option B would be 277 mm
 	vessel_radius_curv = 342.*CLHEP::mm;  //needs to include the vessel thickness, as we construct from outside inwards.
 	vessel_radius = 254.*CLHEP::mm;
-	dist_pmt_vessel = 2*CLHEP::mm;      
+	dist_pmt_vessel = 8*CLHEP::mm;
```

Comment:
Looks like these changes (and below) are only applied in `SetNuPrismShort_mPMTGeometry`; but how should we keep it synced with the other detectors, e.g. `SetNuPrismBeamTest_mPMTGeometry` and later `HyperK_mPMT`? i.e. a single mPMT class that's shared across detectors?

Reply:
I think for that we can change these values similar to `SetNuPrismShort_mPMTGeometry` in all the other detectors. I haven't done much on that but I am sure it will be similar.